src/textual/app.py

```diff
@@ -393,20 +393,12 @@ async def capture_mouse(self, widget: Widget | None) -> None:
             await widget.post_message(events.MouseCapture(self, self.mouse_position))
 
     def panic(self, *renderables: RenderableType) -> None:
-        """Exits the app with a traceback.
+        """Exits the app after displaying a message.
```

Comment:
```suggestion
        """Exits the app and then displays a message.
```

Reply:
"app" makes me think of the process, rather than just the full-screen terminal.